src/server/routes/consents.ts

```diff
@@ -255,6 +263,8 @@ router.get(
     let state = res.locals;
     const sc_gu_u = req.cookies.SC_GU_U;
 
+    // get CMP state from queryParams on redirected request
+    const _cmpConsentedState = isStringBoolean(req.query.cmp as string); // todo code smell
```

Comment:
Could perhaps rename the `isStringBoolean` to `parseBooleanParam`. I would probably just accepts all the potential types from the query string in the method to avoid type casting